deploy/iso/minikube-iso/arch/x86_64/package/containerd-bin/config.toml

```diff
@@ -40,8 +40,7 @@ oom_score = 0
     [plugins."io.containerd.grpc.v1.cri".containerd]
       discard_unpacked_layers = true
       snapshotter = "overlayfs"
-      [plugins."io.containerd.grpc.v1.cri".containerd.default_runtime]
-        runtime_type = "io.containerd.runc.v2"
+      default_runtime_name = "runc"
```

Comment:
Should this also be removed?

https://github.com/kubernetes/minikube/blob/4f3d0da802c7514e57551f42ba6fb9f0abc3d4b0/deploy/iso/minikube-iso/arch/x86_64/package/containerd-bin/config.toml.default#L77

Reply:
as agreed, we'll remove these _config.toml.default_ files (for both x86_64 and aarch64) as they're not used

btw, they can be generated with `containerd config default`